src/servers/graphql-old-server.ts

```diff
@@ -98,16 +99,18 @@ const resolvers = {
       }
     },
     prices: async (_, { length = 365 * 24 * 10 }, { logger }) => {
-      const hourly = await getHourlyPrice({ logger })
+      const prices = await getHourlyPrice({ logger })
+      const currentPrice = await getCurrentPrice()
 
       // adding the current price as the lat index array
       // use by the mobile application to convert prices
-      hourly.push({
+      prices.push({
         id: moment().unix(),
-        o: getCurrentPrice(),
+        o: currentPrice,
```

Comment:
This is better but I don't think it was a bug since this is fed to GraphQL and GraphQL resolvers are okay with a promise as the resolved value.

Reply:
could you please check in your local machine? I was getting the last hour price, if this should work then maybe is something in getCurrentPrice

Reply:
maybe your setup didn't fetch `getCurrentPrice()` because it uses another pod and it was not set up correctly on your machine/docker-compose?